internal/services/eventhub/eventhub_namespace_resource.go

```diff
@@ -471,6 +512,16 @@ func resourceEventHubNamespaceRead(d *pluginsdk.ResourceData, meta interface{})
 			d.Set("maximum_throughput_units", int(*props.MaximumThroughputUnits))
 			d.Set("zone_redundant", props.ZoneRedundant)
 			d.Set("dedicated_cluster_id", props.ClusterArmId)
+			if props.DisableLocalAuth != nil {
+				localAuthEnable := *props.DisableLocalAuth
+				d.Set("local_authentication_enabled", !localAuthEnable)
+			}
+			if props.PublicNetworkAccess != nil {
+				d.Set("public_network_access_enabled", *props.PublicNetworkAccess == namespaces.PublicNetworkAccessEnabled)
+			}
+			if props.MinimumTlsVersion != nil {
+				d.Set("minimum_tls_version", *props.MinimumTlsVersion)
+			}
```

Comment:
all 3 of these fields need to be set all of the time, with a default value if they're not returned e.g.

```
publicNetworkAccessEnabled := true
if props.PublicNetworkAccess != nil {
   *props.PublicNetworkAccess == namespaces.PublicNetworkAccessEnabled
}
d.Set("public_network_access_enabled", publicNetworkAccessEnabled)
```